test/integration/cookies.js

```diff
@@ -118,4 +128,10 @@ describe('Cookies', () => {
             .then(sendRequest);
     });
 
+    describe('Test cookie roles', () => {
+        it('Should contain the studyRoles for user mochatest', () => {
+            parsedCookie.studyRoles.should.deep.equal(studyRolesMock);
```

Comment:
When testing sub-properties, it is a good idea to first test their existence to avoid _undefined has no property `should`_.

`parsedCookie.should.have.property('studyRoles');`